services/admin/src/admin/app.py

```diff
@@ -49,20 +47,27 @@ def create_app() -> Starlette:
 
     init_logging(level=app_config.log.level)
     # ^ set first to have logs as soon as possible
-    assets_directory = init_assets_dir(directory=app_config.assets.storage_directory)
-    cached_assets_directory = init_cached_assets_dir(directory=app_config.cached_assets.storage_directory)
     duckdb_index_cache_directory = init_duckdb_index_cache_dir(directory=app_config.duckdb_index.cache_directory)
     hf_datasets_cache_directory = init_hf_datasets_cache_dir(app_config.datasets_based.hf_datasets_cache)
     parquet_metadata_directory = init_parquet_metadata_dir(directory=app_config.parquet_metadata.storage_directory)
     statistics_cache_directory = init_statistics_cache_dir(app_config.descriptive_statistics.cache_directory)
 
-    if not exists(assets_directory):
-        raise RuntimeError("The assets storage directory could not be accessed. Exiting.")
-
-    if not exists(cached_assets_directory):
-        raise RuntimeError("The cached-assets storage directory could not be accessed. Exiting.")
-
     processing_graph = ProcessingGraph(app_config.processing_graph)
+    cached_assets_storage_client = StorageClient(
```

Comment:
what if we try to access the storage when creating the client, to exit in case of a configuration error, as before (see above: `raise RuntimeError("The assets storage directory could not be accessed. Exiting.")`)

Reply:
I added a very basic validation after client creation (try to list files) in case of errors, a StorageClientInitializeError will be thrown.

Reply:
OK. Just to be sure: will listing files always be quick?

Reply:
Yes, in this case, we will always have only "cached-assets" and "assets" folders so, it will be super fast to get them listed.

Reply:
Another quick validation is simply to to call `fs.exists()` on the directory